project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import Keys._
 
 object Dependencies {
   val Versions = Seq(
-    crossScalaVersions := Seq("2.10.5", "2.11.6"),
+    crossScalaVersions := Seq("2.11.8", "2.11.0"),
```

Comment:
Did you mean to cross compile 2.11 and 2.10?  i.e. `Seq("2.11.8", "2.10.6")`
Minor versions are binary compatible by design.